### ISSUE TITLE
After Cron skills not working

### ISSUE THREAD

**Issue by ramotar** [bumbleshoot/automate-habitica#20]
**What is the bug?**
After Cron skills are not working, if `AUTO_CRON` is disabled.

**Additional context**
Comparing `lastCron` to `lastAfterCron` always leads to a false result, since `let lastAfterCron = new Date(properties["LAST_AFTER_CRON"]);` never gives a valid date object - the only place the property is set is within the `if` clause that is not working due to the property not being set.

**How often does this happen?**
Every time given `AUTO_CRON` is disabled.

**Script version**
Automate Habitica v0.29.32 (beta) by @bumbleshoot

**Script settings**
```javascript
const AUTO_CRON = false; // true or false

const AUTO_ACCEPT_QUEST_INVITES = true;

const FORCE_START_QUESTS = false; // party leaders only
const FORCE_START_QUESTS_AFTER_HOURS = 1; // eg. if set to 1, quests will force start after 1h
const NOTIFY_MEMBERS_EXCLUDED_FROM_QUEST = false;

const AUTO_INVITE_GOLD_QUESTS = false;
const AUTO_INVITE_UNLOCKABLE_QUESTS = false;
const AUTO_INVITE_PET_QUESTS = false;
const AUTO_INVITE_HOURGLASS_QUESTS = false;
const PM_WHEN_OUT_OF_QUEST_SCROLLS = false;

const NOTIFY_ON_QUEST_END = true;

const AUTO_CAST_SKILLS = true;

const AUTO_PAUSE_RESUME_DAMAGE = false;
const MAX_PLAYER_DAMAGE = 20;
const MAX_PARTY_DAMAGE = 5;

const AUTO_ALLOCATE_STAT_POINTS = true;
const STAT_TO_ALLOCATE = "str"; // str, int, con, per

const AUTO_PURCHASE_GEMS = true; // subscribers only

const AUTO_PURCHASE_ARMOIRES = true;
const RESERVE_GOLD = 10000;

const AUTO_SELL_EGGS = false;
const RESERVE_EGGS = 999;

const AUTO_SELL_HATCHING_POTIONS = false;
const RESERVE_HATCHING_POTIONS = 999;

const AUTO_SELL_FOOD = false;
const RESERVE_FOOD = 999;

const AUTO_HATCH_FEED_PETS = true;
const ONLY_USE_DROP_FOOD = true;

const HIDE_PARTY_NOTIFICATIONS = false;
const HIDE_ALL_GUILD_NOTIFICATIONS = false;
const HIDE_NOTIFICATIONS_FROM_SPECIFIC_GUILDS = [ // only used if HIDE_ALL_GUILD_NOTIFICATIONS is false
  "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx", // enter a guild ID between the quotes to hide notifications from that guild
  "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" // repeat for as many guilds as you want
]
```

**Comment by ramotar** [comment]
I also have a fix for this and will submit together with the fix for #19

**Comment by bumbleshoot** [comment]
Hey, thanks so much for catching this! This used to work fine, when the code fetched each script property individually (via [`PropertiesService.getScriptProperties.getProperty(key)`](https://developers.google.com/apps-script/reference/properties/properties#getpropertykey)), but now that it is fetching them all at once (via [`PropertiesService.getScriptProperties.getProperties()`](https://developers.google.com/apps-script/reference/properties/properties#getproperties)), missing script properties are `undefined` rather than `null`. `null` produces a valid Date object, but `undefined` does not!

This has been fixed in v0.29.35. Please [update your script](https://github.com/bumbleshoot/automate-habitica#updating-the-script)! :)
